### PR TITLE
Update SUNBURST/Solarigate compromise

### PR DESCRIPTION
**Pull request by joshuagl**
Update the SUNBURST/Solarigate compromise base on the details from the
CrowdStrike blog on compromsing the publishing infrastructure with SUNSPOT

Signed-off-by: Joshua Lock <jlock@vmware.com>

### CODE REVIEW

File: supply-chain-security/compromises/README.md
Comment (TheFoxAtWork) [inline supply-chain-security/compromises/README.md:18]:
```suggestion
| [SUNBURST/Solarigate](2020/solarwinds.md) | 2020 | Publishing Infrastructure | [1](https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html), [2](https://msrc-blog.microsoft.com/2020/12/13/customer-guidance-on-recent-nation-state-cyber-attacks/), [3](https://www.crowdstrike.com/blog/sunspot-malware-technical-analysis/) |
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
CC @magnologan I think this is is purely publishing infrastructure as the injection occurred right before build.

**Comment by joshuagl** [comment]
> CC @magnologan I think this is is purely publishing infrastructure as the injection occurred right before build.

Agreed. I've force pushed an update to the README and taken the opportunity to improve the commit message a bit.

**Comment by magnologan** [comment]
Kk, got it! LGTM.

**Comment by joshuagl** [comment]
Rebased after the merge of #507.

**Review by TheFoxAtWork — APPROVED** [review]

**Pull request merged**
